modules/api_loader.lua

```diff
@@ -9,73 +9,162 @@ local function get_name(xml_element)
     end
     return name
 end
+ 
 
-local function loadEnums(api, dest)
-  local enums = api:xpath("//interface/enum")
-  for _, e in ipairs(enums) do
-    local enum = { }
-    local i = 1
-    for _, item in ipairs(e:children("element")) do
-      enum[item:attr("name")] = i
-      i = i + 1
+local function dump(o)
+  if type(o) == 'table' then
+    local s = '{ '
+    for k,v in pairs(o) do
+      if type(k) ~= 'number' then k = '"'..k..'"' end
+      s = s .. '['..k..'] = ' .. dump(v) .. ','
     end
-    dest.enum[get_name(e)] = enum
+    return s .. '} \n'
+  else
+    return tostring(o)
```

Comment:
Maybe better to [reorder](https://adc.luxoft.com/confluence/x/kQTyE) this method for readability?